nameko/exceptions.py

```diff
@@ -1,10 +1,15 @@
 from __future__ import unicode_literals
 
-import collections
 import inspect
+import sys
 
 import six
 
+if sys.version_info >= (3, 3):
```

Comment:
Nameko only supports 3.4 upwards (and even 3.4 is being removed in the 3.x release) so I think we can skip this conditional and just keep the updated import.

Reply:
Sorry I'm using 2.12.0 which still supports Python 2, didn't realize master had dropped Python <3.4 support, fixed!

Reply:
We will still support Python 2 in Nameko's 3.x release actually, but are dropping Python 3 versions that are already end of life.

Thanks for the PR!

Reply:
I've put it back as 3.3+ because that's when the feature was introduced

Reply:
Perfect, thank you. Sorry for the unnecessary back and forth